tracks/python/exercises/kindergarten-garden/mentoring.md

```diff
@@ -0,0 +1,36 @@
+### Reasonable solutions
+
+```python
+class Garden(object):
+
+    DEFAULT_STUDENTS = [
+        'Alice', 'Bob', 'Charlie', 'David',
+        'Eve', 'Fred', 'Ginny', 'Harriet',
+        'Ileana', 'Joseph', 'Kincaid', 'Larry',
+    ]
+
+    PLANT_NAMES = {
+        'C' : 'Clover',
+        'G' : 'Grass',
+        'R' : 'Radishes',
+        'V' : 'Violets'
+    }
+
+    def __init__(self, diagram, students=DEFAULT_STUDENTS):
+        self.rows = diagram.split('\n')
```

Comment:
Also, we should probably encourage the use of `diagram.splitlines()`, as that would more portably handle **diagram** values created on OS that do not use the single char `\n` newline convention.